web/app_options.js

```diff
@@ -68,6 +68,11 @@ const defaultOptions = {
     value: 0,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  defaultZoomDelay: {
+    /** @type {number */
+    value: 500,
```

Comment:
This seems like a quite large value, especially since you apply it to *every* zoom operation despite the commit message only talking about mouse-wheel/pinch zooming.

Looking at the changes in `web/app.js` you also (indirectly) apply it to the UI +/- buttons as well as the keyboard shortcuts, was this actually intended here!?

Reply:
I've not looked through all the code, but does that mean that if I zoom *only once* I still have to wait *half a second* for the page to actually start to re-render? Because if so, that seems like it could make the viewer feel quite sluggish; maybe just pick a lower value?

Reply:
In fact when you click only once on the the `+/-`, the rendering after the css zoom is not that bad and the difference between before redraw and after redraw is very tight so I'd say that the user won't see anything.

Reply:
> In fact when you click only once on the the `+/-`, 

That's not the only way to zoom though. Please consider that case where you first zoom to say 10% and then choose e.g. "page-width", the page will now look completely unreadable for *more than half a second*.

Reply:
If I understand correctly, in this case `set currentScaleValue` is used and there is no delay here so it'll behave as before this patch. Or am I missing something ?

Reply:
OK, that might be safe then; however I still think that a value of `500 ms` feels too high.